packages/dx-react-scheduler/src/plugins/scheduler-core.tsx

```diff
@@ -20,9 +20,9 @@ class SchedulerCoreBase extends React.PureComponent<SchedulerProps> {
       >
         <Getter name="appointments" value={appointments(data)} />
         <Getter name="formatDate" value={formatDateTimeGetter(locale)} />
-        <Getter name="layoutHeight" value={height} />
         <Template name="root">
-          <Root>
+          <Root height={height} >
+            <TemplatePlaceholder name="drawer" />
```

Comment:
We should use only common words. `Drawer` -> schedulerContent / schedulerRoot or something else